armsrc/iso14443a.c

```diff
@@ -1390,7 +1389,31 @@ void SimulateIso14443aTag(int tagType, int flags, uint8_t *data) {
         }
     }
 
-    cmd_send(CMD_ACK, 1, 0, 0, 0, 0);
+    Dbprintf("Emulator stopped.  Trace length: %d ", BigBuf_get_traceLen());
+    
+    // print collected ar/nr nonces
+    for (uint8_t i = 0; i < nonces_count; i++) {
+        // for interactive call send collected nonces to client
+        if ((flags & FLAG_INTERACTIVE) == FLAG_INTERACTIVE)
+            cmd_send(CMD_ACK, CMD_SIMULATE_MIFARE_CARD, 0, 0, &ar_nr_collected[i], sizeof(nonces_t));
+        else {
+            // for non-interactive call print nonces
+            Dbprintf("%s, sector %2d: mfkey32v2 %08x %08x %08x %08x %08x %08x %08x",
+                     ar_nr_collected[i].keytype == 0 ? "keyA" : "keyB",
+                     ar_nr_collected[i].sector,
+                     ar_nr_collected[i].cuid,  //UID
+                     ar_nr_collected[i].nonce, //NT
+                     ar_nr_collected[i].nr,    //NR1
+                     ar_nr_collected[i].ar,    //AR1
+                     ar_nr_collected[i].nonce2,//NT2
+                     ar_nr_collected[i].nr2,   //NR2
+                     ar_nr_collected[i].ar2    //AR2
+                    );
+        }
+    }
+    // interrupt interactive client
+    if ((flags & FLAG_INTERACTIVE) == FLAG_INTERACTIVE)
+        cmd_send(CMD_ACK, 1, 0, 0, 0, 0);
```

Comment:
I removed this printing years ago,   I see no reason to bring it back. 
hence the need for flag_interactive is also useless.